espnet2/tasks/diar.py

```diff
@@ -74,6 +82,15 @@
     type_check=AbsDecoder,
     default="linear",
 )
+attractor_choices = ClassChoices(
+    "attractor",
+    classes=dict(
+        rnn=RnnAttractor,
```

Comment:
Please also add a `None` case for models without attractor

Reply:
It seems that we don't explicitly add `None` case in the choices (see https://github.com/espnet/espnet/blob/master/espnet2/tasks/asr.py). Instead, I modified `default="rnn"` to `default=None`.

Reply:
Yeah, that's what I'm expecting! Thanks